benchalerts/tests/integration_tests/test_alert_pipeline.py

```diff
@@ -75,10 +75,16 @@ def test_alert_pipeline(monkeypatch: pytest.MonkeyPatch, github_auth: str):
     # now a real pipeline
     pipeline_steps = [
         steps.GetConbenchZComparisonStep(
-            commit_hash=velox_commit, z_score_threshold=None, step_name="z_none"
+            commit_hash=velox_commit,
+            baseline_run_type=steps.BaselineRunCandidates.fork_point,
```

Comment:
I think this velox commit is on the default branch, so this won't be populated. Maybe choose `parent`.

You can `pytest.skip` this integration test for now until we deploy the latest API changes there.